### DISCUSSION TITLE
thank you

### DISCUSSION THREAD

**Discussion by GamerGirlandCo** [AveYo/MediaCreationTool.bat#162] [General]
i just wanted to show my appreciation for your hard work without needlessly opening an issue. getting this to work must've been no easy task and i am forever indebted to you and everyone else who made this possible :]

**Comment by AveYo** [comment]
Thank you. While it seems like a one-man-army effort, I always valued input from users and tried to make it more easier to use, address particular issues, or try to help with unrelated things (troubleshooting windows update in general). It just that I do well so there's not much need for that (input), ha ha 🥇 

I'm not done. I've been itching to add support for 11 22H2, but it's honestly not there yet. I await a much needed svc refresh or at least a stable online LCU for it. And I want to implement downgrade support. And maybe even ARM support! I think it's bloody time ;)
